cwltool/singularity.py

```diff
@@ -467,7 +467,7 @@ def create_runtime(
         runtime = [
             "singularity",
             "--quiet",
-            "exec",
+            "run",
```

Comment:
```suggestion
            "run",
            "--no-eval",
```

but only for Singularity versions 3.10.0+ and apptainer versions 1.1.0+